draft-ietf-detnet-dp-sol-ip-02.xml

```diff
@@ -1274,11 +1279,175 @@ DetNet                    |L2/SbN|          |L2/SbN|
+          enabled End Systems (hosts), connected to DetNet (DN) enabled
+          IP networks, operating over a DetNet aware MPLS network. In
+          this figure, Relay nodes sit at the boundary of the MPLS
+          domain since the non-MPLS domain is DetNet aware.  This figure
+          is very similar to the DetNet MPLS Network figure in <xref
+          target="I-D.ietf-detnet-dp-sol-mpls"/>.  The primary
+          difference is that the Relay nodes are at the edges of the
+          MPLS domain and therefore function as T-PEs, and that service
+          sub-layer functions are not provided over the DetNet IP
+          network.  The transit node functions show above are identical
+          to those described in <xref
+          target="I-D.ietf-detnet-dp-sol-mpls"/>.
+        </t>
+
+        <t>
+          <xref target="fig_ip_pw_detnet"/> illustrates how relay nodes
+          can provide service protection over an MPLS domain.  In this
+          case, CE1 and CE2 are IP DetNet end systems which are
+          interconnected via a MPLS domain such as described in <xref
+          target="I-D.ietf-detnet-dp-sol-mpls"/>. Note that R1 and R3
+          sit at the edges of an MPLS domain and therefore are similar
+          to T-PEs, while R2 sits in the middle of the domain and is
+          therefore similar to an S-PE.
+        </t>
+
+        <figure align="center" anchor="fig_ip_pw_detnet"
+                title="DetNet IP Over DetNet MPLS Network">
+          <artwork><![CDATA[
+      DetNet                                         DetNet
+IP    Service         Transit          Transit       Service  IP
+DetNet               |<-Tnl->|        |<-Tnl->|               DetNet
+End     |            V   1   V        V   2   V            |  End
+System  |   +--------+       +--------+       +--------+   |  System
++---+   |   |   R1   |=======|   R2   |=======|   R3   |   |   +---+
+|   |-------|._X_....|..DF1..|.__ ___.|..DF3..|...._X_.|-------|   |
+|CE1|   |   |    \   |       |   X    |       |   /    |   |   |CE2|
+|   |   |   |     \_.|..DF2..|._/ \__.|..DF4..|._/     |   |   |   |
++---+       |        |=======|        |=======|        |       +---+
+    ^       +--------+       +--------+       +--------+       ^
+    |        Relay Node       Relay Node       Relay Node      |
+    |          (T-PE)           (S-PE)          (T-PE)         |
+    |                                                          |
+    |<-DN IP-> <-------- DetNet MPLS ---------------> <-DN IP->|
+    |                                                          |
+    |<-------------- End to End DetNet Service --------------->|
+
+   -------------------------- Data Flow ------------------------->
+
+    X   = Service protection (PRF, PREOF, PEF/POF)
+    DFx = DetNet member flow x over a TE LSP
+    ]]></artwork>
+        </figure>
+
+        <figure align="center" anchor="fig_ip_mpls_detnet"
```

Comment:
I am not sure we need this in the IP data plane document. Non-DetNet Aware IP has nothing to do with DetNet. The whole DetNet network is MPLS based. So we say nothing about a DetNet IP Network.

I would propose to delete line 1374-1444.